app/lib/is-js.js

```diff
@@ -2,5 +2,8 @@ export default function isJavascript(scriptTag) {
   // TODO: add a console warning if the script tag doesn't have an attribute? 
   // seems like it's required for some parts of ember consumption
   let type = scriptTag.attributes.type ? scriptTag.attributes.type.value : 'text/javascript';
-  return /(?:application|text)\/(deferred-)?javascript/i.test(type);
+  // guard against chrome-extension:// scripts
+  let badSrc = scriptTag.src && !scriptTag.src.match(/^http|^\//);
```

Comment:
I was just wondering if there would ever be a case where src would be relative to the HTML its being used in, e.g. <script src="./bundle.js">, which would not match either `^http` or `^/`, but would still potentially be valid JS.